src/api/views.py

```diff
@@ -408,4 +408,89 @@ def compare(request):
         else:
             return Response(
                 serializer.errors,status=status.HTTP_400_BAD_REQUEST
-                )
\ No newline at end of file
+                )
+
+@api_view(['GET', 'POST'])
+@renderer_classes((JSONRenderer,))
+def check_license(request):
+    """ Handle License Check API request """
+    if request.method == 'GET':
+        """ Return all license check API request """
+        query = CheckLicenseFileUpload.objects.all()
+        serializer = CheckLicenseSerializer(query, many=True)
+        return Response(serializer.data)
+
+    elif request.method == 'POST':
+        """ Return license check tool result on the post file"""
+        serializer = CheckLicenseSerializer(data=request.data)
+        if serializer.is_valid():
+            if (jpype.isJVMStarted()==0):
+                """ If JVM not already started, start it, attach a Thread and start processing the request """
+                classpath =settings.JAR_ABSOLUTE_PATH
+                jpype.startJVM(jpype.getDefaultJVMPath(),"-ea","-Djava.class.path=%s"%classpath)
+            """ Attach a Thread and start processing the request """
+            jpype.attachThreadToJVM()
+            package = jpype.JPackage("org.spdx.compare")
+            compareclass = package.LicenseCompareHelper
+            try :
+                if request.FILES["file"]:
+                    """ Saving file to the media directory """
+                    myfile = request.FILES['file']
+                    licensetext = myfile.read()
+                    folder = "api/"+str(request.user) +"/"+ str(int(time()))
+                    fs = FileSystemStorage(location=settings.MEDIA_ROOT +"/"+ folder,
+                        base_url=urljoin(settings.MEDIA_URL, folder+'/')
+                        )
+                    filename = fs.save(myfile.name, myfile)
+                    uploaded_file_url = fs.url(filename)
```

Comment:
hey @yugeshk, the above lines of code will save the file twice in the media folder, once in the api folder and then in the apifiles folder, this was a bug which was fixed in #17, so please have a look at that code and try to modify this code accordingly, feel free to ask any doubts slightly_smiling_face